pages/favorites.js

```diff
@@ -2,32 +2,36 @@ import ActivityCard from "@/components/ActivityCard";
 import { StyledList } from "@/styles";
 
 export default function Favorites({
-  activityData,
   favoriteActivitiesList,
   onToggleFavorite,
+  results,
 }) {
-  const favoriteActivities = activityData.filter((activity) =>
+  const favoriteActivities = results.filter((activity) =>
```

Comment:
Are you sure, you want this behaviour? I would expect to find all my favourites at the favourites page.